recipes/DateTimeBasicsLog/src/Main.purs

```diff
@@ -0,0 +1,93 @@
+module DateTimeBasicsLog.Main where
+
+import Prelude
+
+import Data.Date (Date, Month(..), canonicalDate, exactDate)
+import Data.DateTime (DateTime(..), adjust, diff)
+import Data.Enum (toEnum)
+import Data.Maybe (Maybe, fromJust)
+import Data.Time (Hour, Minute, Time(..))
+import Data.Time.Duration as D
+import Effect (Effect)
+import Effect.Console (log)
+import Partial.Unsafe (unsafePartial)
+
+main :: Effect Unit
+main = do
+  log "To create a specific part of a `Time` value, you must use \
```

Comment:
In the places where we need to long strings like this in our code at work we'll usually use `"""` as in:

```purs
"""
To create a specific part of a `Time` value, you must use the `toEnum`
function. This ensures the value you provide is within the correct bounds.
"""
```

which saves having to add all the `\`. Not necessary, but thought I'd share it in case it helps with writing some of these long strings.

Reply:
That's a good point. I'd prefer that style.